lib/rides/pages/ride_detail_page.dart

```diff
@@ -113,13 +113,30 @@ class _RideDetailPageState extends State<RideDetailPage> {
       final Ride ride = _ride!;
 
       final Profile driver = ride.drive!.driver!;
-      widgets.add(ProfileWidget(driver, showDescription: true, withHero: true));
-      widgets.add(const Divider(thickness: 1));
-
-      widgets.add(ReviewsPreview(driver));
-
-      if (driver.profileFeatures!.isNotEmpty) widgets.add(const Divider(thickness: 1));
-      widgets.add(FeaturesColumn(driver.profileFeatures!));
+      widgets.addAll(<Widget>[
+        Align(
+          alignment: Alignment.centerLeft,
+          child: Text(S.of(context).pageRideDetailDriver, style: Theme.of(context).textTheme.titleLarge),
+        ),
```

Comment:
I think this one is unnecessary, the driver is already on the ride card and there we don't need to explain it either. But maybe put the divider below this row instead?

Reply:
Actually let's just add space everywhere instead of dividers?

Reply:
1) This is a detail page, so there may be a bit more detail for people who don't understand the profile on the card. 
2) I think the profile widget really doesn't have the context to make it work like on the cards, where there is only the most vital information displayed. Here, e.g. the riders are also displayed and there is much more information surrounding the widget, so I think it's less clear: If every rider gets a widget, maybe this belongs to the reviews right below? You might very well still guess correctly, but there are more possibilities here. 
3) It adds consistency and the widget felt strange being the only one without a title (except for the overview). Especially since the section "riders" has a title, "driver" should too.
4) It doubles as a title to the whole section marked by the dividers, where everything is details about the driver, including the reviews and features. This is in response to the user studies, where some people thought e.g. the features were your features. This is also why I don't think removing the divider below this is a good idea.

Reply:
I thought we agreed that a well-designed app does not really need dividers, because things are presented and spaced in such a way that it guides your sight and understanding of how everything fits together.

Reply:
But dividers help guide your sight :D BlaBlaCar also has dividers and I thought we agreed that was a very well-designed app. I think dividers can be very useful to group things together, and as long as they are not overused, I don't think they help clarity